src/main/java/de/tum/in/www1/artemis/service/AutomaticTextFeedbackService.java

```diff
@@ -30,11 +32,25 @@
 
     private static final double DISTANCE_THRESHOLD = 1;
 
+    // TODO: Update this value later
+    private static final double LAMBDA_THRESHOLD = 3;
+
     private final TextBlockRepository textBlockRepository;
 
-    public AutomaticTextFeedbackService(FeedbackService feedbackService, TextBlockRepository textBlockRepository) {
+    private final TreeNodeRepository treeNodeRepository;
+
+    private final PairwiseDistanceRepository pairwiseDistanceRepository;
```

Comment:
Codacy found an issue: [Perhaps 'pairwiseDistanceRepository' could be replaced by a local variable.](https://app.codacy.com/gh/ls1intum/Artemis/pullRequest?prid=5968857)